superbit_lensing/galsim/mock_superBIT_data.py

```diff
@@ -1003,14 +1110,18 @@ def main():
             dt = time.time() - start
             logprint(f'Total time for star injections: {dt:.1f}s')
 
+
         else:
             # get local range to iterate over in this process
             local_start, local_end = M.mpi_local_range(sbparams.nstars)
             for k in range(local_start, local_end):
+                print(f'doing star {k}')
```

Comment:
Any prints should be in the `logprint()` both for logging purposes and to speed up code if you don't want it printed to terminal. Also I think this print is already included in the various `make_obj` functions

Reply:
This print statement was useful for quick debugging while the code was being written, but is no longer necessary. It has been removed.